.github/workflows/rebase-linux-nvidia-6.12.yml

```diff
@@ -0,0 +1,57 @@
+name: Update and Rebase linux-nvidia-6.12
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  update-rebase-branch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout current branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref_name }}
+          fetch-depth: 0
+
+      - name: Add upstream remote (if not exists)
+        run: |
+          if ! git remote | grep -q upstream; then
+            git remote add upstream https://github.com/gregkh/linux.git
+          fi
+
+      - name: Fetch upstream tags
+        run: git fetch upstream --tags
+
+      - name: Find latest stable tag for branch
+        id: latest_tag
+        run: |
+          tag=$(git tag -l 'v6.12.*' --sort=-v:refname | head -1)
```

Comment:
We can probably use the following to get the latest upstream tag in a more generic way without hard-coding v6.12:
```
version="${{ github.ref_name#linux-nvidia-}}"
tag=$(git describe --abbrev=0 --tags upstream/linux-${version}.y)
```